homeassistant/components/schlage/lock.py

```diff
@@ -0,0 +1,65 @@
+"""Platform for Schlage lock integration."""
+from __future__ import annotations
+
+from typing import Any
+
+from pyschlage import Schlage
+from pyschlage.lock import Lock
+
+from homeassistant.components.lock import LockEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, MANUFACTURER
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Schlage WiFi locks based on a config entry."""
+    api: Schlage = hass.data[DOMAIN][config_entry.entry_id]
+    locks: list[Lock] = await hass.async_add_executor_job(api.locks)
+    async_add_entities([SchlageLock(lock) for lock in locks])
+
+
+class SchlageLock(LockEntity):
+    """Schlage lock entity."""
+
+    def __init__(self, lock: Lock) -> None:
+        """Initialize a Schlage Lock."""
+        super().__init__()
+        self._lock: Lock = lock
```

Comment:
Please use the new entity naming scheme

https://developers.home-assistant.io/blog/2022/07/10/entity_naming/

Reply:
Done